app/src/main/res/values-fr/strings.xml

```diff
@@ -28,7 +28,7 @@
     <string name="stopped">Arrêté</string>
 
     <string name="create_backup_dir_failed">Echec de la création du répertoire de sauvegarde</string>
-    <string name="create_msg_log_failed">Échec de la création du fichier du journal des messages</string>
+    <string name="create_msg_log_failed">Echec de la création du fichier du journal des messages</string>
 
     <string name="create_backup_dir">Création du répertoire de sauvegarde</string>
     <string name="allow_storage_permission_msg">Les messages et les médias ne peuvent pas être enregistrés sans accès au stockage. Redémarrez l'application et autorisez la permission.</string>
```

Comment:
Concerning **l'application** the apostrophe should be escaped or it will give build error

solution: **l\\'application**